### PR TITLE
Simplify PHP installation

### PR DESCRIPTION
**Pull request by stanley-cheung**
* This should be the final piece to issue #4517 "PHP installation is too complex"
 * Updated `README.md`. Got rid of `libgrpc-dev` and `homebrew` as dependencies. Now in both linux and mac, all we need is `sudo pecl install grpc-beta`
 * Bump the version to 0.9.0 for the next release

### CODE REVIEW

**Comment by stanley-cheung** [comment]
This is for master. PHP Basic tests passed. Interop tests passed. Artifacts test continue to pass: https://grpc-testing.appspot.com/view/Artifacts/job/gRPC_distribtest/222/

**Comment by jtattermusch** [comment]
@ctiller , does this qualify as mergeable this week? It doesn't change any executable code so it should be fine.

File: package.xml [hunk 1]
```diff
@@ -10,11 +10,11 @@
   <email>grpc-packages@google.com</email>
   <active>yes</active>
  </lead>
- <date>2016-02-24</date>
+ <date>2016-03-01</date>
  <time>16:06:07</time>
  <version>
-  <release>0.8.0</release>
-  <api>0.8.0</api>
+  <release>0.9.0</release>
```
Comment (jtattermusch) [inline package.xml]: With many languages we've skipped a few version numbers to stay consistent with core release. Seems like we should do that for PHP as well?

**Comment by ctiller** [comment]
I'm not against it, but it's on that tricky borderline (build.yaml changes).

**Comment by nicolasnoble** [comment]
I'm not against it either. There's no direct build.yaml file change actually.

File: package.xml (hunk 1 above)
Comment (stanley-cheung) [inline package.xml]: Sounds fair to me. The package has come along far enough. I can update it to 0.14 for the next one.

Comment (stanley-cheung) [inline package.xml]: Fixed

**Comment by jtattermusch** [comment]
LGTM. Merge when green.

**Comment by ctiller** [comment]
It's probably worth unifying the version number with the version block at
the top of build.yaml when you update to 0.14: it'll make our lives
slightly easier in the future.

**Comment by stanley-cheung** [comment]
Pecl doesn't seem to like a version like `0.14.0-dev`. I will figure this out next time.

**Comment by stanley-cheung** [comment]
PHP tests are green. Merging

**Pull request merged**